router.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const Router = require('koa-router');
+const router = new Router;
+
+const spotifyService = require('./service-providers/spotify-service');
+const duetifyService = require('./service-providers/duetify-service');
+
+router
+  .get('/login', spotifyService.login)
```

Comment:
That's a good point. I'll note that for something to perhaps update, sooner rather than later.